04-data-structures-part1.Rmd

```diff
@@ -43,6 +43,7 @@ is.numeric() # is it numeric?
 is.integer() # is it an integer?
 is.complex() # is it complex number data?
 is.character() # is it character data?
+str()  # what is it?
```

Comment:
There's also `class()` -- but I think of `str()` and `class()` as more general interrogation functions than just for atomic types and it might get a tad confusing. E.g.:
```r
> a <- 1
> str(a)
 num 1
> class(a)
[1] "numeric"
> typeof(a)
[1] "double"
```

Is this the best place to have them introduced in the lesson? I definitely agree they should be in there somewhere though, and this indeed might be the best place to do it.